meson_options.txt

```diff
@@ -65,3 +65,8 @@ option('try_static_libs',
        choices : ['opusfile', 'png', 'sdl2', 'sdl2_net'],
        value : [],
        description : 'Attempt to statically link selected libraries.')
+
+option('use_gtest',
+       type : 'feature',
+       value : 'auto',
+       description : 'Build unit tests')
```

Comment:
I think I made a mistake in asking for rename to `use_gtest`. All `use_*` options we have get value 'false'/'true' (and one gets 'auto'). Maybe we'll transition to use option type feature for them, but not for as long as we use 2 buildsystems.

Let's change name of this option again, to: `unit_tests`. This way user will be configuring it using `-Dunit_tests=enabled`, `-Dunit_tests=disabled`; and it will be in line with description.